deckhouse-controller/cmd/deckhouse-controller/main.go

```diff
@@ -64,11 +65,13 @@ const (
 func main() {
 	sh_app.Version = ShellOperatorVersion
 	ad_app.Version = AddonOperatorVersion
+	ParseFilePath := strings.Split(os.Args[0], "/")
```

Comment:
Consider using filepath module for file path operations: https://pkg.go.dev/path/filepath#Base

Reply:
Thank you for note